x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/notice.ts

```diff
@@ -1,29 +1,61 @@
 /* eslint-disable @kbn/eslint/require-license-header */
 
 /* @notice
+ * Detection Rules
```

Comment:
Any objections to these updates? Looks like we might need a `.rstrip()` on each line?

Reply:
no - looks like that is just how the txt file is organized. But we do the `toml` vs `json` is a problem. Would it be ok to just strip the extension from the txt file and leave it out of both